.github/workflows/test-download.yaml

```
@@ -0,0 +1,52 @@
+name: Tests (Download)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  tests:
+
+    strategy:
+      fail-fast: false  # continue other tests if one test in matrix fails
+      matrix:
+        node-version: [16.x]
+        os: [ubuntu-latest]
+
+    name: Test kit on Node v${{ matrix.node-version }} (${{ matrix.os }})
+    continue-on-error: true
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 25
+
+    defaults:
+      run:
+        shell: bash
+
+    env:
+      CYPRESS_CACHE_FOLDER: ~/.cache/Cypress
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node v${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        cache: 'npm'
+        node-version: '16'
```

If we're using a matrix this should match the version in the matrix

```suggestion
        node-version: ${{ matrix.node-version }}
```